static/game/bomb.js

```diff
@@ -133,12 +134,13 @@ export class Bomb extends Entity {
       const radius =
         (Math.abs(this.fuseTime) / this.timeToExplode) * this.blastRadius;
       const thickness = (radius / this.blastRadius) * 60;
-      circle(
+      centeredRect(
         this.drawPos,
-        radius,
+        radius * 2,
+        radius * 2,
         undefined,
-        thickness,
-        `hsl(${this.hue}, 100%, 50%)`
+        `hsl(${this.hue}, 100%, 50%)`,
+        thickness
       );
```

Comment:
Not sure I like the bomb explosions being square. I think it would be better to have circular bombs that deal damage and destroy blocks in a circle (see #29)